B/blis/common.jl

```diff
@@ -56,19 +61,19 @@ function blis_script(;blis32::Bool=false)
             export BLI_CONFIG=firestorm
```

Comment:
```suggestion
           export BLI_CONFIG=arm64
```
There is an armsve config upstream actually, so I wonder if it would be better for us to do arm64 here as well (yes, I know we would be compiling a lot of kernels we wouldn't need), but I think it could be good to have the kernel autodetection to use SVE when available.